src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -14,7 +15,7 @@
 /*
 * These values should be kept in sync with System.Globalization.CalendarId
 */
-enum CalendarId : int32_t
+enum CalendarId : int16_t
```

Comment:
Yes, it needs to match the managed side, since it is getting marshalled back in the GetCalendars method.  The managed side has the enum defined as 'ushort'.